src/Classes/ImportTab.lua

```diff
@@ -628,6 +628,17 @@ function ImportTabClass:ImportItem(itemData, slotName)
 		end
 	else
 		item.name = itemLib.sanitiseItemText(itemData.typeLine)
```

Comment:
Looks like this ends up being "Superior Energy Blade" if there's quality on the gem.  We could use `itemData.baseType` instead, but I'm not sure what else that would affect, as far as the typeLine goes